### PR TITLE
Moving workflows library into cpg_utils

### PR DESCRIPTION
**Pull request by vladsavelyev**
Another part of moving the workflows library into cpg_utils, after the cpg_utils PR: https://github.com/populationgenomics/cpg-utils/pull/63

### CODE REVIEW

**Comment by lgruen** [comment]
@vladsavelyev Did you want to assign subsets of this PR to different reviewers?

**Comment by vladsavelyev** [comment]
I don't think it's realistic, there are too many changes even for 3 reviewers. I'm looking into breaking it up into multiple PRs

File: jobs/query_scripts/vep.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
+"""
```
[the remark this answers is not in the document]
Comment (vladsavelyev) [reply jobs/query_scripts/vep.py:1]: But they are in different folders! `stages/vep.py` is a workflow stage, `jobs/vep.py` is a making batch jobs. Would you suggest to make the structure more flat?

File: jobs/fastqc.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        fname = basename(str(input_path))
+        if isinstance(input_path, FastqPair):
+            if not str(input_path).endswith('.gz'):
+                cmd += f"""\
+                mv {input_file} {input_file}.gz
+                """
+                input_file = f'{input_file}.gz'
+                fname += '.gz'
+            if subsample:
+                n = 10_000_000
```
[the remark this answers is not in the document]
Comment (vladsavelyev) [reply jobs/fastqc.py:54]: Yep, that's the only reason. Should probably move the constant to the module level, or even make it customaisable with the config!

File: configs/seqr.toml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    'CPG255232',  # perth-neuro: new
+    'CPG255240',  # perth-neuro: new
+    'CPG253328',  # perth-neuro, contamination rate 32%
+    'CPG13409',   # perth-neuro, coverage ~0x
+    'CPG243717',  # validation, NA12878_KCCG low coverage https://main-web.populationgenomics.org.au/validation/qc/cram/multiqc.html,
+    'CPG246645',  # ag-hidden, eof issue  https://batch.hail.populationgenomics.org.au/batches/97645/jobs/440
+    'CPG246678',  # ag-hidden, diff fastq size  https://batch.hail.populationgenomics.org.au/batches/97645/jobs/446
+    'CPG246561',  # ag-hidden, coverage ~0x https://main-web.populationgenomics.org.au/ag-hidden/qc/cram/multiqc.html
+]
+
```
[the remark this answers is not in the document]
Comment (vladsavelyev) [reply configs/seqr.toml:35]: Yeah, that's the plan to handle that with metamist

**Pull request closed without merging**